Common++/header/PointerVector.h

```diff
@@ -343,21 +372,16 @@ namespace pcpp
 		static std::vector<T*> deepCopyUnsafe(std::vector<T*> const& origin)
```

Comment:
Maybe we can change the method name to just `deepCopy()`?

Reply:
It is specifically named unsafe as the returned vector contains allocated objects that need to be manually freed. same with `freeVectorUnsafe` where the pointers in the vector refer to invalid values after the function return.